src/utils/GitHubUtils/GitHubUtils.js

```diff
@@ -28,88 +28,45 @@ const getUserSvg = async (proxyServerUrl, gitHubUsername) => {
   };
 };
 
-export const setEmptyCalendarValues = calendar => {
-  const copiedCalendar = javaScriptUtils.deepCopyObject(calendar);
-
-  copiedCalendar.attributes = {
+export const initialize = produce(draft => {
+  draft.attributes = {
     viewBox: "0 0 828 128",
     preserveAspectRatio: "xMidYMin slice",
     class: "js-calendar-graph-svg",
     style:
       "width: 100%; padding-bottom: 14.5%; height: 1px; overflow: visible; font-size: .8em"
   };
 
-  copiedCalendar.children[0].children.forEach((weeklyData, weekIndex) => {
-    weeklyData.children.forEach((dailyData, dayIndex) => {
-      copiedCalendar.children[0].children[weekIndex].children[
-        dayIndex
-      ].attributes = {
-        ...dailyData.attributes,
-        "data-count": "0",
-        fill: "#ebedf0"
-      };
-    });
-  });
-
-  copiedCalendar.children[0].children.forEach(data => {
-    if (data.name === "text") {
-      // eslint-disable-next-line no-param-reassign
-      data.attributes = { ...data.attributes, fill: "#767676" };
+  draft.children[0].children.forEach((week, weekIndex) => {
+    if (week.name === "text") {
+      draft.children[0].children[weekIndex].fill = "#767676";
     }
-  });
 
-  return copiedCalendar;
-};
-
-export const mergeCalendarsContributions = (
-  actualCalendar,
-  gitHubUserJsonCalendar,
-  timeframe
-) => {
-  const copiedActualCalendar = javaScriptUtils.deepCopyObject(actualCalendar);
-
-  gitHubUserJsonCalendar.children[0].children.forEach(
-    (weeklyData, weekIndex) => {
-      weeklyData.children
-        .filter(dailyData =>
-          calendarUtils.filterContributionDays(dailyData, timeframe)
-        )
-        .forEach((dailyData, dayIndex) => {
-          const actualCalendarDailyData = calendarUtils.getCalendarDataByIndexes(
-            copiedActualCalendar,
-            weekIndex,
-            dayIndex
-          );
-          const totalDailyContributions =
-            Number(actualCalendarDailyData.attributes["data-count"]) +
-            Number(dailyData.attributes["data-count"]);
-
-          copiedActualCalendar.children[0].children[weekIndex].children[
-            dayIndex
-          ].attributes = {
-            ...actualCalendarDailyData.attributes,
-            "data-count": String(totalDailyContributions),
-            fill: calendarUtils.getFillColor(totalDailyContributions)
-          };
-        });
-    }
-  );
+    week.children.forEach((day, dayIndex) => {
+      draft.children[0].children[weekIndex].children[dayIndex].attributes[
+        "data-count"
+      ] = 0;
+      draft.children[0].children[weekIndex].children[dayIndex].attributes.fill =
+        "#ebedf0";
```

Comment:
Reason for this diff: I don't remember off the top of my head why, but we iterated over the calendar twice (now there's only a single, nested iteration).